src/validatePrTitle.js

```diff
@@ -54,11 +58,13 @@ module.exports = async function validatePrTitle(
     );
   }
 
-  if (scopes && result.scope && !scopes.includes(result.scope)) {
+  const givenScopes = result.scope ? result.scope.split(',') : undefined;
```

Comment:
Maybe trim the individual parts to allow for `fix(core, ui):`?

```suggestion
  const givenScopes = result.scope ? result.scope.split(',').map(scope => scope.trim()) : undefined;
```

Reply:
💯